index.html

```diff
@@ -1049,6 +1049,395 @@ <h2>Evidence</h2>
+that uniquely identifies a subject">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "ProofOfAgeCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01",
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"name": "Mr John Doe",
+		"address": "10 Some Street, Anytown, ThisLocal, Country X",
+		"dateOfBirth": "2001-01-01",
+		"gender": "male",
+		"placeOfBirth": "Anytown, ThisLocal, Country X",
+		"nationality": "X",
+	},
+	"proof": { ...
+	}
+}
+        </pre>
+
+
+   </section>
+
+
+   
+   <section>
+      <h2>Holder Acts on Behalf of Subject</h2>
+
+      <p>
+When the holder wants to act on behalf of the subject, the issuer MAY indicate this by
+either including a relationship object in the subject's <a>claim</a>, or by issuing a separate relationship credential to the holder. Either way, the relationship object is used to
+describe the relationship between the holder and the subject of the claim. </p>
+
+<p>It is outside
+the scope of this standard how the issuer verifies the subject, the holder and the
+relationship between them.
+
+<p>
+Note: it is for further study which, if any, of the above mechanisms is to be 
+standardised by the working group.
+
+     </p>
+
+   <section>
+
+      <h3>The Relationship Object</h3>
+
+      <p>
+The Relationship object states the relationship between a subject, identified by an ID, and an object, also identified by an ID e.g. a value of "Parent" indicates that the subject is the parent of the object. 
+     </p>
+      <p>
+Note that values of the relationship type will be defined in other standards, 
+but it is 
+likely that typical values might be "Parent" (when the subject is a child) and 
+"Owner" (when the subject is a pet) etc.
+      </p>
+
+<p>
+Note that the following text and the two examples have not yet been finalised
+by the working group.
+</p>
+<p>
+When the relationship object is contained
+within a subject's claim, then the holder is the object of the
+relationship and the claim's subject is the subject of the relationship. When the
+relationship property is contained in the holder's relationship credential, the holder is the subject of both the relationship credential and the relationship, and the 
+relationship object refers to the subject of one or more claims or verifiable credentials that are held by the holder.
+
+      </p>
+ 
+     
+        <pre class="example nohighlight" title="An example of the relationship 
+property in a child's credential, so that a parent can hold it">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "ProofOfAgeCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01T19:73:24Z",
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"ageUnder": 16,
+		"relationship": {
+			"type": "Child",
+			"id": "did:example:ebfeb1c276e12ec211f712ebc6f"
```

Comment:
Which is the child and which is the parent here? It seems like "relationship" should read "parent":

```js
{
   "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
   "ageUnder": 16,
   "parent": "did:example:ebfeb1c276e12ec211f712ebc6f"
}
```

That would say that the subject (`did:example:ebfeb1f712ebc6f1c276e12ec21`) is under 16 and has a parent identified by `did:example:ebfeb1c276e12ec211f712ebc6f`.

Reply:
My answer above answers this question of yours.
The difference in style is that I have created a generic relationship object that can be used in different places, and the value of the type is not fixed but will be defined by external specifications. You have created a property rather than an object, but the property is unknown as it will be specified by external specifications.

Reply:
The generic relationship property has no semantics on its own so I don't see it as useful. It also is a break from the graph model where properties establish meaningful links (relationships with semantics) between nodes in the graph.

All you need to do is define a property -- you don't need the extra baggage of a type and a meaningless relationship object.

Reply:
Either way you are going to have to define a new type:
- either a new type of VC which contains a new property, e.g. grandson, or
- a new type of relationship in the relationship object, with relationship being a standard property of the generic VC type.